js/controller.js

```diff
@@ -441,6 +445,10 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         this.state.screenToShow = CONSTANTS.SCREEN.END_SCREEN;
         this.mb.publish(OO.EVENTS.END_SCREEN_SHOWN);
       }
+      if (Utils.isIPhone()){
+        //iPhone end screen is the same as start screen, except for the replay button
+        this.state.screenToShow = CONSTANTS.SCREEN.START_SCREEN;
```

Comment:
@aldigun, for iPhone we always show start screen for everything. We have a special check that looks if the start screen is shown when the playerState == CONSTANTS.STATE.END, and then instead of play button we show replay button.